### PR TITLE
Update eslint config for cypress plugin

### PR DESCRIPTION
**Pull request by Prasannams97**

### CODE REVIEW

**Comment by Prasannams97** [comment]
@yedhink _a Please review this PR.

File: .eslintrc.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ module.exports = {
     // custom rules cannot override the following rules.
     "./.eslint-rules/imports/enforced",
     "./.eslint-rules/react",
-    "./.eslint-rules/promise"
+    "./.eslint-rules/promise",
+    "plugin:cypress/recommended"
```
Comment (yedhink) [inline .eslintrc.js:29]: Move this to the top, that is to line 15.

**Review by yedhink — COMMENTED** [review]
@Prasannams97 _a

I have added some comments. Please go through them and let me know if you need any clarifications.

**Comment by Prasannams97** [comment]
@yedhink _a Please check.

**Comment by yedhink** [comment]
@Prasannams97 _a

Few things:
- Please don't name branches with mixed cases like it has been named in this PR. Always use lowercase alpha-numeric characters separated by hyphens.
- When a review comment is fixed in your local system, resolve it, if no further queries are there as part of that review comment.

Now here:
- Add the cypress eslint plugin to the end of [this line](https://github.com/bigbinary/wheel/blob/dc2513dde99a304d14c7005aedcf4f3ee88a82ab/.scripts/sync_with_wheel.sh#L13) in that script. Make sure that `yarn add -D ...` command works after passing this new package as arg to it.
- Once done, raise an issue in https://github.com/bigbinary/bigbinary-website/issues asking to install this new package, and assign that issue to me.

**Comment by Prasannams97** [comment]
> @Prasannams97 _a
> 
> Few things:
> 
> * Please don't name branches with mixed cases like it has been named in this PR. Always use lowercase alpha-numeric characters separated by hyphens.
> * When a review comment is fixed in your local system, resolve it, if no further queries are there as part of that review comment.
> * Raise an issue first. Then name the branch with issue number and then work on that branch for the PR that fixes that particular issue. That's the BigBinary way of making a PR. In the PR's first comment we need to explicitly mention `Fixes #xxx` where `xxx` needs to be replaced with the issue number.
> 
> Now here:
> 
> * Add the cypress eslint plugin to the end of [this line](https://github.com/bigbinary/wheel/blob/dc2513dde99a304d14c7005aedcf4f3ee88a82ab/.scripts/sync_with_wheel.sh#L13) in that script. Make sure that `yarn add -D ...` command works after passing this new package as arg to it.
> * Once done, raise an issue in https://github.com/bigbinary/bigbinary-website/issues asking to install this new package, and assign that issue to me.

-  Added cypress eslint plugin to script. `yarn add -D ...` command is working as I pass this new package as an argument to it.
- Raised an issue https://github.com/bigbinary/bigbinary-website/issues/1592#issue-1011703842 and assigned it to you.

**Comment by Prasannams97** [comment]
@yedhink _a

**Review by yedhink — APPROVED** [review]

**Pull request merged**